packages/docs/src/components/theme-toggle/README.md

```diff
@@ -0,0 +1,3 @@
+# Theme switch
+
+Based on: [GUI challenges - Theme Toggle](https://github.com/argyleink/gui-challenges/tree/main/theme-switch)
```

Comment:
I hope we can use this? (I rewritten this to qwik, but logic, animation, css are almost the same)

Reply:
video before a few fixes, but still shows the toggle
https://www.loom.com/share/8d88f7f3b6fb4c9a9e95e9e8fc85f31c